src/main/views/edit-request.njk

```diff
@@ -0,0 +1,446 @@
+          <div id="reason-form-group">
+            <input class="govuk-input govuk-input--width-10" id="reason-input" name="reason" type="text" placeholder="Reason">
+          </div>
+        </td>
+        <td class="govuk-table__cell" style="text-align: right;">
+          <a class="govuk-link--no-visited-state" style="margin-right: 0.5rem"  href="#new-edit-reference-row"  id="submit-button" onclick="submitEditReference()">Save<span class="govuk-visually-hidden"> edit reference</span></a>
+        </td>
+        <td class="govuk-table__cell">
+          <a class="govuk-link--no-visited-state" href="#new-edit-reference-row" onclick="cancelEditReference()">Cancel<span class="govuk-visually-hidden"> edit reference</span></a>
+        </td>
+      </tr>
+        {% for instruction in editRequest.edit_instructions %}
+        <tr id="instruction-{{ loop.index0 }}" class="govuk-table__row">
+          <td class="govuk-table__cell"> {{ instruction.start_of_cut }}</td>
+          <td class="govuk-table__cell"> {{ instruction.end_of_cut }}</td>
+          <td class="govuk-table__cell"> {{ instruction.difference }}</td>
+          <td class="govuk-table__cell"> {{ instruction.reason }}</td>
+          <td class="govuk-table__cell" style="text-align: right;">
+            <a class="govuk-link" style="margin-right: 0.5rem"  href="#new-edit-reference-row" onclick="selectRow({{ loop.index0 }}, true)">Update<span class="govuk-visually-hidden"> edit reference</span></a>
+          </td>
+          <td class="govuk-table__cell">
+            <a class="govuk-link" href="#delete-reference-row" onclick="selectRow({{ loop.index0 }}, false)">Delete<span class="govuk-visually-hidden"> edit reference</span></a>
+          </td>
+        </tr>
+        {% endfor %}
+    </tbody>
+  </table>
+  </form>
+
+  <div style="text-align: right;">
+    <button id="submit-button" class="govuk-button" data-module="govuk-button" onclick="onSubmit()" data-testid="">Submit</button>
+  </div>
+
+  <style>
+    tr.delete-message-block {
+      & > td > div.delete-message-block {
+        border-left: 5px solid #d4351c;
+        padding-left: 15px;
+
+        & > p {
+          margin: 0;
+          color: #d4351c;
+        }
+      }
+
+      input:disabled {
+        border-color: #d4351c;
+        opacity: 1;
+        color: #00000080;
+      }
+    }
+  </style>
+
+  <script type="text/javascript" crossorigin="anonymous" data-testid="recording-video-script">
+    const videoContainer = document.getElementById("video-container");
+    var videoError = document.getElementById("videoError");
+
+    const playerConfig = {
+        key: "{{ mediaKindPlayerKey }}",
+        ui: true,
+        playback: {
+            muted: false,
+            autoplay: false,
+            preferredTech: [{ player: 'html5', streaming: 'hls'}]
+        }
+    };
+
+    let player = new mkplayer.MKPlayer(videoContainer, playerConfig);
+
+    fetch("{{ recordingPlaybackDataUrl }}", {
```

Comment:
part of this url is from the request params I think I saw in the route? need to make sure it's sanitised as it's essentially user input so can't be trusted?